torch/testing/_internal/common_nn.py

```diff
@@ -3791,6 +3797,33 @@ def fractional_max_pool3d_test(test_case):
             ),
         )
 
+# Check that non linear activations work with no batch dimensions
+non_linear_activations_no_batch = [
+    'ELU', 'Hardshrink', 'Hardsigmoid', 'Hardtanh', 'Hardswish', 'LeakyReLU',
+    'LogSigmoid', 'PReLU', 'ReLU', 'ReLU6', 'RReLU', 'SELU', 'CELU', 'GELU',
+    'Sigmoid', 'SiLU', 'Mish', 'Softplus', 'Softshrink', 'Softsign', 'Tanh',
+    'Tanhshrink', 'Threshold'
+]
+non_linear_activations_extra_info = {
+    'CELU': {'constructor_args': (2.,)},
+    'Threshold': {'constructor_args': (2., 1.)},
+    'Hardsigmoid': {'check_gradgrad': False},
+    'Hardswish': {'check_gradgrad': False},
+    'RReLU': {'test_cuda': False},
```

Comment:
Since the RRelu is random and the random generation is different between CPU and GPU, this fails the test that checks the CPU output is the same as the GPU output:

https://github.com/pytorch/pytorch/blob/357c4d9cc40f461fe7b95af6524bbf8b2ecd9f74/torch/testing/_internal/common_nn.py#L5122

Reply:
Cool, thanks!